icons/xd/xd-line.svg

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 21.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
```

Comment:
Maybe cleanup those comments and other unused lines (like Layer_2 at the end)?

Reply:
Maybe yes, I am not an expert in svg. It would be great if you could help!

Reply:
It's quite simple, just remove the lines ;-)